### DISCUSSION TITLE
Data Aquisition using Telemed MicrUs Ultrasound probe in Ubuntu 20.04

### DISCUSSION THREAD

**Discussion by mistimohan** [PlusToolkit/PlusLib#1084] [Q&A]
I am completely new to Plus Library and the data acquisition process of ultrasound data .So I have few doubts regarding use of Plus library .
I have a telemed MicrUs EXT-1H [https://www.telemedultrasound.com/micrus/?lang=en](https://www.telemedultrasound.com/micrus/?lang=en)
And I am able to find that PlusLib supports Telemed MicrUs probe [here](http://perk-software.cs.queensu.ca/plus/doc/nightly/dev/classTelemedUltrasound.html) 
My whole project is based on ROS noetic ,Ubuntu 20.04 so it is necessary to use Ubuntu system.
There is a developer mode option for building PlusLib from source for Linux based systems.
But In the Documentation of PlusLibrary,it has been mentioned that :
"Unfortunately, many of the drivers written for devices are Windows specific, and thus capture cannot be done on a Linux or MacOSX machine. It is recommended to do the data acquisition on Windows and stream the acquired data to the Linux or MacOS computer for further processing."
I want to confirm:
1)can we build PlusLib for ubuntu 20.04 (as documentation provides info about only ubuntu 16.04)
2) Does Building PlusLib from source enables me to capture ultrasound images using telemed probe or not ?
If not ,then i request you to provide step by step guidance on how to get ultrasound image from PlusLib ,windows 10 and stream the acquired data to Ubuntu ? What are the possible options to get forward with this situation?

**Comment by ungi** [comment]
Hi, my understanding is that MicrUs ultrasound can only be used on Windows. You don't need to build PLUS, because PLUS with Telemed support is already available as an installer for Windows. If your application is on a Linux computer, then you need a separate Windows computer for the ultrasound, and you can stream the images (and ultrasound control commands) via network between the two computers.

**Comment by mistimohan** [comment]
Thank you for quick response.

**Comment by MarkWijkhuizen** [comment]
Is there an update on this issue, did Linux support for Telemed got added in the meantime?
We have the same situation currently where we need to capture and process frames from the Telemed device on a Linux system.

**Comment by mistimohan** [comment]
As @ungi responded, I am using a separate Windows computer for ultrasound and transferring data to a Linux PC.
